include/spi/ITimer.h

```diff
@@ -26,7 +26,7 @@ class ITimerVisitor {
 	virtual void trigger(ITimer* triggeringTimer) = 0;
 };
 
-typedef std::function<void (ITimer* triggeringTimer)> TimerCallback;
+using TimerCallback = std::function<void (ITimer* triggeringTimer)>;
```

Comment:
Why "using" here and "typedef" at other places (like [here](https://github.com/Openwide-Ingenierie/libezsp/blob/bb17749c44dcc6853d2d85ef7d6d7a75944e963d/src/ezsp/lib-ezsp-main.h#L41-L43))?